.github/label-actions.yml

```
@@ -27,3 +27,75 @@
 
     The Renovate team will take a look at the reproduction repository.
     Once we confirm the provided repository reproduces the problem, the label will be changed to `reproduction:confirmed`.
+
+'logs:problem':
+  comment: >
+    Hi there,
+
+
+    We have found that there's a problem with the logs.
+    Depending on which situation applies follow one, some or all of these instructions.
+
+
+    ### No logs at all
+
+    If there's no log posted yet, we need you to find and copy/paste the log into the issue template.
+
+
+    #### Finding logs on hosted app
+
+    <details><summary>Click me to read instructions</summary>
+
+
+    If you use the Renovate app (GitHub/GitLab/etc):
```

```suggestion
    If you use the Renovate app (GitHub):
```

Maybe we should strip GitLab, as that is currently only self-hosted?

---

Agreed